### PR TITLE
chore: upgrade eslint-doc-generator v1.4

### PR DESCRIPTION
**Pull request by aladdin-add**

### CODE REVIEW

**Comment by aladdin-add** [comment]
I used `npm run update:eslint-docs` to update the docs, but it still shows a few errors:
```bash
➜  eslint-plugin-node git:(master) npm run update:eslint-docs

> eslint-plugin-n@15.6.0 update:eslint-docs
> eslint-doc-generator --config-emoji recommended-module,☑️ --config-emoji recommended-script,✔️ --rule-list-split meta.docs.category --ignore-config recommended-module --ignore-config recommended-script --url-configs "https://github.com/eslint-community/eslint-plugin-n#-configs"

`no-extraneous-import` rule doc should have included: include
`no-extraneous-import` rule doc should have included: exclude
`no-extraneous-import` rule doc should have included: replace
`no-extraneous-require` rule doc should have included: include
`no-extraneous-require` rule doc should have included: exclude
`no-extraneous-require` rule doc should have included: replace
```

@bmish can you please take a look, thanks! :)

**Comment by aladdin-add** [comment]
I used `npm run update:eslint-docs` to update the docs, but it still report a few errors:
```bash
➜  eslint-plugin-node git:(master) npm run update:eslint-docs

> eslint-plugin-n@15.6.0 update:eslint-docs
> eslint-doc-generator --config-emoji recommended-module,☑️ --config-emoji recommended-script,✔️ --rule-list-split meta.docs.category --ignore-config recommended-module --ignore-config recommended-script --url-configs "https://github.com/eslint-community/eslint-plugin-n#-configs"

`no-extraneous-import` rule doc should have included: include
`no-extraneous-import` rule doc should have included: exclude
`no-extraneous-import` rule doc should have included: replace
`no-extraneous-require` rule doc should have included: include
`no-extraneous-require` rule doc should have included: exclude
`no-extraneous-require` rule doc should have included: replace
```

@bmish can you take a look, thanks!

**Comment by bmish** [comment]
I just investigated and it looks like this is correct. eslint-doc-generator is correctly detecting additional options that are undocumented. We have a TODO in the doc to mention these options.

https://github.com/eslint-community/eslint-plugin-n/blob/master/docs/rules/no-extraneous-require.md?plain=1#L60-L62

https://github.com/eslint-community/eslint-plugin-n/blob/master/docs/rules/no-extraneous-import.md?plain=1#L52-L54

The options come from a shared schema:

https://github.com/eslint-community/eslint-plugin-n/blob/master/lib/util/get-convert-path.js#L167-L183

I would recommend just adding a bullet-point list to add some initial documentation for these options:

* `include`: TODO
* `exclude`: TODO
* `replace`: TODO

File: docs/rules/no-extraneous-import.md [hunk 1]
```diff
@@ -51,7 +51,9 @@ Default is `[]`
 
 #### convertPath
 
-TODO
+* `include`: TODO
+* `exclude`: TODO
+* `replace`: TODO
```
Comment (bmish) [inline docs/rules/no-extraneous-import.md:56]: alphabetical and dash

```suggestion
- `exclude`: TODO
- `include`: TODO
- `replace`: TODO
```

File: docs/rules/no-extraneous-require.md [hunk 2]
```diff
@@ -59,7 +59,9 @@ Default is `[".js", ".json", ".node"]`.
 
 #### convertPath
 
-TODO
+* `include`: TODO
+* `exclude`: TODO
+* `replace`: TODO
```
Comment (bmish) [inline docs/rules/no-extraneous-require.md:64]: alphabetical and dash

```suggestion
- `exclude`: TODO
- `include`: TODO
- `replace`: TODO
```

**Review by bmish — APPROVED** [review]

**Pull request merged**